### PR TITLE
Tag for wasm check job

### PR DESCRIPTION
**Pull request by TriplEight**
- runs `check-web-wasm:` job on `ci1-docker` or `ci2-docker` runners.

### CODE REVIEW

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -140,12 +141,12 @@ build-rust-doc-release:            &build
...
-  <<:                              *build-only
+  # <<:                              *build-only
   tags:
     - linux-docker
-  before_script:
-    - sccache -s
-    - ./scripts/build.sh
+  # before_script:
+  #   - sccache -s
+  #   - ./scripts/build.sh
```
Comment (gabreal) [inline .gitlab-ci.yml]: these can probably be remove entirely, don't remember why it was put there.

File: .gitlab-ci.yml [hunk 2]
```diff
@@ -140,12 +141,12 @@ build-rust-doc-release:            &build
     expire_in:                     7 days
     paths:
     - ./crate-docs
-  <<:                              *build-only
+  # <<:                              *build-only
```
Comment (gabreal) [inline .gitlab-ci.yml]: if there is no only statement the doc build will run every time. If that is desired why not have it consistent with *test-linux-stable*?

File: .gitlab-ci.yml (hunk 1 above)
Comment (TriplEight) [reply .gitlab-ci.yml]: I included my experiment in this branch accidentially, sorry, it should't be there.
This job won't work without building WASM binaries.

File: .gitlab-ci.yml (hunk 2 above)
Comment (TriplEight) [reply .gitlab-ci.yml]: it was just for the test run, won't be in the final commit.

File: .gitlab-ci.yml (hunk 1 above)
Comment (gabreal) [reply .gitlab-ci.yml]: also hadn't tested it before.
```
error: couldn't read node/executor/src/../../runtime/wasm/target/wasm32-unknown-unknown/release/node_runtime.compact.wasm: No such file or directory (os error 2)
  --> node/executor/src/lib.rs:26:100
   |
26 | native_executor_instance!(pub Executor, node_runtime::api::dispatch, node_runtime::native_version, include_bytes!("../../runtime/wasm/target/wasm32-unknown-unknown/release/node_runtime.compact.wasm"));
   |                                                                                                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

error: aborting due to previous error

error: Could not compile `node-executor`.
```
but doesn't work without it.

**Pull request merged**